tests/indices/tier/warm.yaml

```diff
@@ -0,0 +1,38 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test endpoints relevant to tiering of index to warm.
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  - path: /games
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create an indices named `books` and `games` with default settings.
+    path: /{index}
+    method: PUT
+    parameters:
+      index: books,games
+
+  - synopsis: Check if the indices `games` and `books` exists. They should.
+    path: /{index}
+    method: HEAD
+    parameters:
+      index: books,games
+      allow_no_indices: true
+      expand_wildcards: none
+      flat_settings: true
+      include_defaults: true
+      ignore_unavailable: true
+      local: true
+
+  - synopsis: Tier the `books` and `games` indices from hot to warm.
+    version: '>= 2.16'
+    path: /{index}/_tier/warm
+    method: POST
+    parameters:
+      index: books,games
+    response:
+      status: 200
```

Comment:
Is there a way to know what tier my index is in?

Reply:
that will be available via _cat/indices later or _cat/tiering api, changes to which are not done yet. This test can be updated once its available.